app/code/Magento/PaypalGraphQl/composer.json

```diff
@@ -13,7 +13,8 @@
         "magento/module-quote-graph-ql": "*",
         "magento/module-sales": "*",
         "magento/module-payment": "*",
-        "magento/module-store": "*"
+        "magento/module-store": "*",
+        "magento/module-customer": "*"
```

Comment:
This can be avoided based on the first comment.

Reply:
**Done**